src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+* {
+  box-sizing: border-box;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: transform 0.3s;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin: auto;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__code {
+    margin: 4px 0 0;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-top: 16px;
+  }
+
+  &__rating-value {
+    margin: 0;
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-top: 24px;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price-title {
+    margin: 0;
+    padding: 0;
+    font-size: 12px;
+    color: $secondary-color;
+  }
+
+  &__price-value {
+    margin: 0;
+    padding: 0;
+    font-size: 16px;
+    font-weight: 700;
+    color: $main-color;
+  }
+
+  &__link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    align-content: center;
```

Comment:
There is no need to add flex property for link, just add the correct text-align and line-height properties to center the text

Reply:
Done